qiskit_ibm_runtime/qiskit/primitives/base_estimator.py

```diff
@@ -278,7 +278,9 @@ def __call__(
             circuits: the list of circuit indices or circuit objects.
             observables: the list of observable indices or observable objects.
             parameter_values: concrete parameters to be bound.
-            run_options: runtime options used for circuit execution.
+            run_options: runtime options used for circuit execution. These include shots,
+                qubit_lo_freq, meas_lo_freq, schedule_los, rep_delay, init_qubits,
+                and use_measure_esp.
```

Comment:
These base classes are copied from terra, let's not update these. just sampler.py and estimator.py should be good enough for this PR.